src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurEventNotificationProcessingServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import java.util.Collection;
+
+import org.apache.commons.collections.CollectionUtils;
+
+import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationProcessingService;
+import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
+import edu.cornell.kfs.concur.businessobjects.ConcurEventNotification;
+import edu.cornell.kfs.concur.rest.xmlObjects.ExpenseDetailedReportDTO;
+import edu.cornell.kfs.concur.service.ConcurAccountValidationService;
+import edu.cornell.kfs.concur.service.ConcurEventNotificationService;
+
+public class ConcurEventNotificationProcessingServiceImpl implements ConcurEventNotificationProcessingService {
+    
+    protected ConcurEventNotificationService concurEventNotificationService;
+    protected ConcurAccountValidationService concurAccountValidationService;
+
+    @Override
+    public void processConcurEventNotifications() {
+        Collection<ConcurEventNotification> concurEventNotifications = concurEventNotificationService.retrieveConcurEventNotificationsForProcessing();
+
+        if (concurEventNotifications != null && CollectionUtils.isNotEmpty(concurEventNotifications)) {
```

Comment:
Since the body of the for loop won't be executed if the collection is empty, you could eliminate the CollectionUtils.isNotEmpty check. Also, you could use ObjectUtils.isNotNull for the null check.